hasura/graphql-engine
# server: hasura on PG v13

### Description

Hasura on Postgres v13

### Changelog


### Affected components



### Related Issues

closes https://github.com/hasura/graphql-engine/issues/5836

### Steps to test and verify

1. Download and load the latest build image [here](https://199925-137724480-gh.circle-artifacts.com/0/server/image.tar).
2. Setup the docker env. 
```docker
version: "3.6"
services:
  postgres:
    image: postgres:13
    restart: always
    volumes:
      - db_data:/var/lib/postgresql/data
    environment:
      POSTGRES_PASSWORD: postgrespassword

  graphql-engine:
    image: hasura/graphql-engine:pull6070-e25c6e6a # replace with the correct image name/tag
    ports:
      - "8090:8080"
    depends_on:
      - "postgres"
    restart: always
    environment:
      HASURA_GRAPHQL_DATABASE_URL: postgres://postgres:postgrespassword@postgres:5432/postgres
      HASURA_GRAPHQL_ENABLE_CONSOLE: "true"
      HASURA_GRAPHQL_DEV_MODE: "true"
      HASURA_GRAPHQL_ENABLED_LOG_TYPES: startup, http-log, webhook-log, websocket-log, query-log

volumes:
  db_data:
```
3. Run `docker exec -it *name of container* pg_dump --version`
to see: `pg_dump (PostgreSQL) 13.0 (Debian 13.0-1.pgdg90+1)`

4. Additionally, you can visit the console on port `8090` to see if everything is working.
